regex-parsing/src/test/resources/finders/MultipleWhitespaceFinder.yml

```diff
@@ -0,0 +1,7 @@
+---
+- 'Hello,   world!'         # Noncompliant {{Replace spaces with quantifier `{2}`.}}
+- 'Hello,  world!'          # Noncompliant {{Replace spaces with quantifier `{1}`.}}
+- 'Hello, world!   '        # Noncompliant
+
+- 'Hello, {3}world!'        # Compliant with quantifier
```

Comment:
minor: Why not have a compliant test with single white spaces separated by non-white space,
e.g.: "Hello , world"